README.md

```diff
@@ -39,8 +38,6 @@ jobs:
     steps:
       - name: Create k8s Kind Cluster
         uses: helm/kind-action@v1.0.0-alpha.3
-        with:
-          install_local_path_provisioner: true
```

Comment:
why removing? did not get it

Reply:
Because the local path provisioner is not installed and the storage class is therefore not available.

Reply:
Doesn't that happen here https://github.com/helm/kind-action/blob/master/kind.sh#L175 ?

Reply:
No. The pr changes the default to not install the local path provisioner as it's not necessarily needed anymore.

Before the test also installed it explicitly what I've removed too.